superset-frontend/src/views/CRUD/data/database/DatabaseList.test.jsx

```diff
@@ -139,6 +139,10 @@ describe('DatabaseList', () => {
     expect(wrapper.find(SubMenu)).toExist();
   });
 
+  it('renders a SubMenu with no tabs', () => {
+    expect(wrapper.find(SubMenu).props().tabs).toBeUndefined();
```

Comment:
is undefined correct here or null?

Reply:
undefined is correct since nothings should be passed to it, not even null.